htgr/htr-10/steady/htr-10-critical.i

```diff
@@ -0,0 +1,289 @@
+
+# State ------------------------------------------------------------------------
+# see state on xs library
+# Power ------------------------------------------------------------------------
+
+# ==============================================================================
+# GLOBAL PARAMETERS
+# ==============================================================================
+[GlobalParams]
+  coupled_flux_groups = 'sflux_g0 sflux_g1 sflux_g2 sflux_g3 sflux_g4 sflux_g5 sflux_g6 sflux_g7 sflux_g8 sflux_g9'
+  scalar_flux = 'sflux_g0 sflux_g1 sflux_g2 sflux_g3 sflux_g4 sflux_g5 sflux_g6 sflux_g7 sflux_g8 sflux_g9'
+  library_file = '../data/xs/htr-10-XS.xml'
+  library_name = 'htr-10-critical'
+  grid_names = 'default'
+  grid = '1'
+  plus = 1
+  isotopes = 'pseudo'
+  densities = '1.0'
+[]
+
+# ==============================================================================
+# GEOMETRY AND MESH
+# ==============================================================================
+[Mesh]
+  [./fmg]
+    type = FileMeshGenerator
+    file = '../data/mesh/htr-10-critical-a-rev6.e'
+    exodus_extra_element_integers = 'eqv_id material_id'
+  [../]
+  [./eqvid]
+    type = ExtraElementIDCopyGenerator
+    input = fmg
+    source_extra_element_id = eqv_id
+    target_extra_element_ids = 'equivalence_id'
+  [../]
+  # These modifiers are used to remove the boronated bricks that surround the core
+  # from the original mesh
+  [./delete_bricks]
+    type = BlockDeletionGenerator
+    input = eqvid
+    block = '3 4 5'
+  [../]
+  [./sideset_side]
+    type = ParsedGenerateSideset
+    input = delete_bricks
+    combinatorial_geometry = 'sqrt(x*x+y*y) > 165.0'
+    new_sideset_name = 1
+  [../]
+  [./sideset_bot]
+    type = ParsedGenerateSideset
+    input = sideset_side
+    combinatorial_geometry = 'z < 41'
+    included_subdomain_ids = '10 11 12'
+    normal = '0 0 -1'
+    new_sideset_name = 2
+  [../]
+  [./sideset_top]
+    type = ParsedGenerateSideset
+    input = sideset_bot
+    combinatorial_geometry = 'z > 490'
+    included_subdomain_ids = '52 45 46'
+    normal = '0 0 1'
+    new_sideset_name = 3
+  [../]
+[]
+
+[Equivalence]
+  type = SPH
+  transport_system = diff
+  compute_factors = false
+  equivalence_grid_names = 'default'
+  equivalence_grid = '1'
+  equivalence_library = '../data/sph/htr-10-Diff-SPH.xml'
+[]
+
+# ==============================================================================
+# VARIABLES AND KERNELS
+# ==============================================================================
+[Variables]
+[]
+
+[Kernels]
+[]
+
+# ==============================================================================
+# AUXVARIABLES AND AUXKERNELS
+# ==============================================================================
+[AuxVariables]
+  [./AbsorptionRR]
+    order = FIRST
+    family = MONOMIAL
+    block = '10 11 12 13 14 15 16 17 18 20 21 22 30 31 32 40 41 42 43 44 45 46 50 52'
```

Comment:
can you use variable substitutions instead? eg ${fuel_blocks}
These listings are not very user friendly